### PR TITLE
Refactoring SDK

### PR DESCRIPTION
**Pull request by real-felix**
SDK refactored to let a place for the Solana SDK

### CODE REVIEW

File: sdk/tbrv3/solana/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "module": "CommonJS",
+    "moduleResolution": "Node",
+    "target": "es6",
+    "resolveJsonModule": true,
+    "esModuleInterop": true
+  },
+  "include": [
+    "src",
+    "../../../target/idl",
+    "index.ts"
```
Comment (scnale) [inline sdk/tbrv3/solana/tsconfig.json:20]: I think this should be included in the `files` property instead.

Comment (real-felix) [reply sdk/tbrv3/solana/tsconfig.json:20]: Done

File: package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "typescript": "5.5.4"
   },
   "workspaces": [
-    "sdk"
+    "sdk/tbrv3/evm",
```
Comment (solanoepalacio) [inline package.json:20]: do we need the `tbrv3` directory here? ultimately this is the only contract we'll have in this repo, isn't it?

File: sdk/tbrv3/solana/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "moduleResolution": "Node",
+    "target": "es6",
+    "resolveJsonModule": true,
+    "esModuleInterop": true
+  },
+  "files": [
+    "src",
+    "../../../target/idl",
+    "index.ts"
+  ]
```
Comment (scnale) [inline sdk/tbrv3/solana/tsconfig.json:21]:
```suggestion
  "include": [
    "src",
    "../../../target/idl"
  ],
  "files": [
    "index.ts"
  ]
```

File: package.json (hunk 2 above)
Comment (scnale) [reply package.json:20]: For now, yeah. But I'm expecting the TBRv3 sdk to outlive the TBRv3 contract and I would like to host all sdks here :thinking:

Comment (solanoepalacio) [reply package.json:20]: gotcha, thanks

**Review by solanoepalacio — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (real-felix) [reply package.json]: It's done as discussed

File: sdk/evm/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "lib": [
+      "es2023"
+    ]
+  },
+  "include": [
+    "tbrv3",
+    "tbrv3/index.ts"
```
Comment (scnale) [inline sdk/evm/tsconfig.json:10]:
```suggestion
    "tbrv3"
```

**Review by scnale — APPROVED** [review]

**Pull request merged**